packages/craco/scripts/build.js

```diff
@@ -20,6 +20,17 @@ const context = {
 loadCracoConfigAsync(context).then(cracoConfig => {
     context.paths = getCraPaths(cracoConfig);
 
+    const { isSupported, version } = getReactScriptVersionInformation(cracoConfig);
```

Comment:
I don't think this code should be in a function which load the config. I would move this to a file, something like `validate-cra-version.js`